diagnostic.md

```diff
@@ -21,13 +22,24 @@ Write the `curl` request you'd use to retrieve a list of all donuts on the
 server.
 
 ```sh
-# your answer here
+curl --include --request "GET" "http://www.example.com/donuts" \
+  --header "Content-Type: application/json" \
+  --data "{
```

Comment:
In a `GET` request, we don't need headers or data. It would just be:
`curl --include --request GET "http://www.example.com/donuts"`